OrbitGl/FrameTrackOnlineProcessor.cpp

```diff
@@ -0,0 +1,50 @@
+#include "FrameTrackOnlineProcessor.h"
+
+#include <limits>
+#include "CaptureData.h"
+#include "TimeGraph.h"
+#include "UserDefinedCaptureData.h"
+
+FrameTrackOnlineProcessor::FrameTrackOnlineProcessor(const CaptureData& capture_data,
+                                                     TimeGraph* time_graph)
+    : time_graph_(time_graph) {
+  const UserDefinedCaptureData& user_defined_capture_data =
+      capture_data.user_defined_capture_data();
+  for (const auto& function : user_defined_capture_data.frame_track_functions()) {
+    const uint64_t function_address = capture_data.GetAbsoluteAddress(function);
+    current_frame_track_functions_.insert(function_address);
+    previous_timestamp_ns_.insert(
+        std::make_pair(function_address, std::numeric_limits<uint64_t>::max()));
+  }
+}
+
+void FrameTrackOnlineProcessor::ProcessTimer(const orbit_client_protos::TimerInfo& timer_info,
+                                             const orbit_client_protos::FunctionInfo& function) {
+  uint64_t function_address = timer_info.function_address();
+  if (current_frame_track_functions_.empty() ||
```

Comment:
If you think that this shortcut is really helpful in terms of performance, I'd add a comment about that.

Reply:
No, this was just an experiment. I don't think it makes a meaningful difference. Thanks for spotting!